app-pda/libplist/libplist-1.12.recipe

```diff
@@ -0,0 +1,80 @@
+SUMMARY="A library to handle Apple Property List format in binary or XML"
+DESCRIPTION="libplist is a library for reading and writing \
+the Apple binary and XML property list format. It is part of \
+the libimobiledevice stack providing access to iDevices."
+HOMEPAGE="http://www.libimobiledevice.org/"
+COPYRIGHT="2009 Nikias Bassen"
+LICENSE="GNU GPL v3"
```

Comment:
It seems libplist has 2 licenses: **`GNU GPL v2`** (according to file **`COPYING`**) and **`GNU LGPL v2.1`** (according to file **`COPYING.LESSER`**), so the recipe should have this:
```
LICENSE="GNU GPL v2
	GNU LGPL v2.1"
```

Regarding the COPYRIGHT variable, could you change it to:
```
COPYRIGHT="2008-2010 Jonathan Beck
	2010-2014 Nikias Bassen"
```
Thanks again!